src/classes/QueryBuilder.cls

```diff
@@ -182,19 +267,26 @@ public without sharing class QueryBuilder extends NebulaCore implements IQueryBu
 
             this.queryFields.add(commonFieldName.toLowerCase());
         }
-        Logger.addEntry(this, 'this.queryFields=' + this.queryFields);
     }
 
     private void addFieldSetMembers() {
         if(this.fieldSet == null) return;
 
-        for(Schema.FieldSetMember field : this.fieldSet.getFields()) this.queryFields.add(field.getFieldPath().toLowerCase());
+        for(Schema.FieldSetMember field : this.fieldSet.getFields()) {
```

Comment:
Can we smoosh this back down to 1 line?
`for(Schema.FieldSetMember field : this.fieldSet.getFields()) this.queryFields.add(field.getFieldPath().toLowerCase());`